client/flutter/lib/components/loading_indicator.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class LoadingIndicator extends StatelessWidget {
```

Comment:
Since this produces a Sliver,  `SliverLoadingIndicator` would be more fitting.

Reply:
In this context I think of it as *our* loading indicator for the app, not a particular style of loading indicator.  We may change its impl or styling from here...